syscalls-gen/src/main.rs

```diff
@@ -15,7 +15,7 @@ use regex::Regex;
 static LINUX_REPO: &str = "https://raw.githubusercontent.com/torvalds/linux";
 
 /// Linux version to pull the syscall tables from.
-static LINUX_VERSION: &str = "v5.17";
+static LINUX_VERSION: &str = "v6.2";
```

Comment:
Did you do a `cargo run` after bumping this? I would expect at least a few new syscalls to have been added since the last update. If not, I'd keep it at the old value.

Reply:
i did, but not a single one was added - i was surprised as well